types/metascraper-address/package.json

```diff
@@ -7,7 +7,7 @@
     ],
     "dependencies": {
         "@types/node": "*",
-        "metascraper": ">=5.14.8 <=5.40.0"
+        "metascraper": "^5.14.8"
```

Comment:
Shouldn’t these all be `"^5.41.0"`? Otherwise `metascraper.Rules` won’t be defined.